web/swoopstyle/html/5-10go/index.html

```diff
@@ -0,0 +1,59 @@
+   <!-- Text Alignment  -->
```

Comment:
where is your doctype, html, head, body ?

Reply:
I haven't finished this one yet! :) currently testing everything 👍